internal/pkg/gateway/config.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package gateway
+
+// GatewayOptions is used to configure the gateway settings
+type GatewayOptions struct {
+	// GatewayEnabled is used to enable the gateway service
+	GatewayEnabled bool
+}
+
+// GetDefaultConfig gets a GatewayOptions with the default
+// Gateway configuration
+func GetDefaultConfig() GatewayOptions {
```

Comment:
Is something like this ok @sykesm ?

Reply:
Yes. It addresses my concern about package state and is fine.

If you want my really picky feedback,
- `gateway.GatewayOptions` is a stuttering name and could simply be `gateway.Options`
- accessors in go, in general, forego the `Get` prefix so this would be `func DefaultOptions() Options { ... }`. In context, callers would use `gateway.DefaultOptions()` to acquire the defaults.

Reply:
Feedback is good :)
That naming looks much better, thanks